automated/build.sh

```diff
@@ -14,7 +20,7 @@ if [ $SHORTNAME = "lts" ]
 then
     TAG=$(echo $TARGET | sed 's@^lts-\([0-9]*\)\.[0-9]*@lts\1@')
     WORKDIR=$ROOT/work/$(echo $TARGET | sed 's@^lts-\([0-9]*\)\.[0-9]*@lts-\1@')
-    if [ -n "$NOPLAN" ]; then
+    if [ -n "${NOPLAN:-}" ]; then
```

Comment:
My ignorance but why is -n "$NOPLAN" considered unsafe actually?

Reply:
Simple if the script reaches this line and `NOPLAN` isn't set it stops:
```
./build.sh: line 17: NOPLAN: unbound variable
```
I just encountered this while trying to build next lts.

As a workaround I start with 😄 
```bash
NOPLAN= ./build.sh lts-22.9
```

```suggestion
    if [[ -v NOPLAN ]]; then
```

Reply:
Oops I guess that is with `-e`, TIL, thanks!!